lib/components/user/places/place-editor.tsx

```diff
@@ -67,18 +84,49 @@ function makeLocationFieldLocation(favoriteLocation: UserSavedLocation) {
 class PlaceEditor extends Component<Props> {
   static contextType = ComponentContext
 
-  _handleLocationChange = (e: LocationSelectedEvent) => {
-    const { setTouched, setValues, values } = this.props
-    const { lat, lon, name } = e.location
+  _setLocation = (location: Location) => {
+    const { intl, setTouched, setValues, values } = this.props
+    const { category, lat, lon, name } = location
     setValues({
       ...values,
-      address: name,
+      address:
+        // If the raw current location is passed without a name attribute (i.e. the address),
+        // set the "address" as the formatted coordinates of the current location at that time.
+        category === 'CURRENT_LOCATION'
+          ? intl.formatMessage({ id: 'common.coordinates' }, { lat, lon })
+          : name,
```

Comment:
To confirm grabbing the current location worked, it could be good to append the current location string so the users sees `41.9746601, -87.6636681 (current location)`. Alternatively you may be able to remove the `map.currentLocation` i18n string since I don't think it gets used anywhere else.

Reply:
I don't think I want to show the text "current location" because the favorite place will not necessarily be the current location when the user travels. Thoughts?

Reply:
> 

I'd agree with this, I'd say the only time I'd show the text "current location" is when the user clicks this button in the location field. 
![image](https://github.com/opentripplanner/otp-react-redux/assets/115499534/46b1c75d-3228-45bc-8c9c-c719964eddf6)

But I don't mind disregarding this idea, I think having the coordinates populate and the map marker is probably enough confirmation that the location was fetched.